### ISSUE TITLE
ECONNREFUSED with node-sass version 3.11.1

### ISSUE THREAD

**Issue by ISO-JBA** [sass/node-sass#1787]
- NPM version (`npm -v`): 2.5.1
- Node version (`node -v`): v0.12.1
- Node Process (`node -p process.versions`): 
{ http_parser: '2.3',
  node: '0.12.1',
  v8: '3.28.73',
  uv: '1.0.2',
  zlib: '1.2.8',
  modules: '14',
  openssl: '1.0.1m' }

- Node Platform (`node -p process.platform`): win32
- Node architecture (`node -p process.arch`): x64
- node-sass version (`node -p "require('node-sass').info"`): error
- npm node-sass versions (`npm ls node-sass`):
 └─┬ grunt-sass@1.2.0
      └── node-sass@3.11.1

Hello,

I experienced some problems with the newest version 3.11.1 of node-sass. We have a jenkins set up on an vm which builds our project. Our build fails because of the following:

> Start downloading binary at https://github.com/sass/node-sass/releases/download/v3.11.1/linux-x64-46_binding.node
Cannot download "https://github.com/sass/node-sass/releases/download/v3.11.1/linux-x64-46_binding.node": 
connect ECONNREFUSED 192.30.253.113:443
Hint: If github.com is not accessible in your location
      try setting a proxy via HTTP_PROXY, e.g. 
      export HTTP_PROXY=http://example.com:1234
or configure npm proxy via
      npm config set proxy http://example.com:8080

The proxy is configured on the vm and i can also download the source via wget. After adding a shrinkwrap file to our project where I define node-sass version 3.10.1 everything was fine again.
But of course I dont want to keep using the old version as a permanent solution.

I'm not sure if that has something to do with the problem but our NO_PROXY environment variable is set, so that the proxy itself would also fall under the rules.

**Comment by djusv** [comment]
I was also having this issue also but I did some quick digging and I think the warning message is a bit confusing. The message states :

2016-11-03 21:53:34.347 | Hint: If github.com is not accessible in your location
2016-11-03 21:53:34.347 | try setting a proxy via HTTP_PROXY, e.g. 
2016-11-03 21:53:34.347 | 
2016-11-03 21:53:34.348 | export HTTP_PROXY=http://example.com:1234
But the system environment proxies are being completely ignored. Looking at merged code from #1725.

So people who before who were using system environment proxies <3.11.0 and now are upgrading to >=3.11.x will encounter this issue (timeout). I experienced this when updating gulp-sass.
